chore: core can vote RFC proposals (#42)
withastro/.github: GOVERNANCE.md

This document outlines the governance model for Astro. This includes detailed descriptions of different roles, nomination processes, code review processes, and Code of Conduct enforcement.

> 👉 **All community members must follow the [Code of Conduct (CoC)](CODE_OF_CONDUCT.md).**

Reviewer: The double spaces at the end of lines here were meaningful (they get rendered as a line break) and shouldn’t be removed:
```suggestion
👉 **All community members must follow the [Code of Conduct (CoC)](CODE_OF_CONDUCT.md).**  
```

Author: I am going to add a line a line break instead, otherwise formatters go nuts

Consequences for CoC violations are detailed in [Moderation](#moderation).

> 👉 **Want to trigger a vote, nomination, or perform some other action?**

Reviewer: Same here:
```suggestion
👉 **Want to trigger a vote, nomination, or perform some other action?**  
```

Astro features are discussed using a model called [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making). This model attempts to achieve consensus on all significant changes to Astro, but has a fallback voting procedure in place if consensus appears unattainable.

A RFC async meeting is held on ad-hoc basis by Core Contributors and TSC to decided which proposals are accepted

1. Anyone can submit an RFC to suggest changes to Astro.
2. A trivial change can be discussed and approved entirely within the RFC GitHub issue, as long as there are no objections from Core or TSC members. This is not considered a formal vote.

> 3. A non-trivial, significant change should be discussed within the RFC and approved during an RFC async meeting. In some cases, an RFC may be approved outside of an RFC meeting using Pull Request reviews as a proxy for votes.
> 4. During an RFC async meeting, the person leading the meeting will attempt to achieve consensus on the RFC proposal.

Reviewer 2: ```suggestion
3. A non-trivial, significant change should be discussed within the RFC and approved during an RFC discussion (either as a meeting call or timed asynchronous discussion thread). In some cases, an RFC may be approved outside of an RFC discussion using Pull Request reviews as a proxy for votes.
4. During an RFC discussion, the discussion leader will attempt to achieve consensus on the RFC proposal.
```

Kinda unclear to me what an "async meeting" means. Isn't the point of a meeting that... everyone is synchronously attending? 😅 

If the goal is to add flexibility to hold a discussion thread in lieu of a call, we should make that more explicit

> Astro features are discussed using a model called [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making). This model attempts to achieve consensus on all significant changes to Astro, but has a fallback voting procedure in place if consensus appears unattainable.

Reviewer: This line here mentions “a fallback voting procedure in place if consensus appears unattainable”. But below there isn’t any detail of this procedure and with the proposed change also not clear who would vote. Should we remove this line about “fallback voting”?

Reviewer 3: I believe this is the fallback being referenced: https://github.com/withastro/.github/pull/42/files#diff-b60c6a93e9f74ee52e71bf33781c2870b8395c8c5a34f00f61305343cb8d8447R371

In the past this was a much more complicated system but we never ended up using it so IIRC we just simplified it to this new line.

> **Who can vote:** All [TSC](#technical-steering-committee-tsc) members.

Reviewer 4: With this line gone entirely, now I'm not sure it ever states anywhere among whom consensus is achieved? (Or I missed it?) Anyone who happens to be on the call? (And, these calls used to be public, so that would include public/maintainers equally?)

Maybe this means clarifying below in number 4, something like:

> 4. During an RFC meeting, the person leading the call will attempt to achieve consensus on the RFC proposal among all attending Astro (L??) maintainers .

Astro features are discussed using a model called [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making). This model attempts to achieve consensus on all significant changes to Astro, but has a fallback voting procedure in place if consensus appears unattainable.

> **Who can vote:** All [TSC](#technical-steering-committee-tsc) members and Core members.

Reviewer 5: Do we need to mention TSC at all since it's a subset of core?

Author: I don't want to take responsibility for adding more changes, as this could lead to a longer discussion.

Reviewer 6: I feel like we can remove this line entirely, the list below this doesn't mention there even being a vote. There really *isn't* a vote.

> **Who can vote:** All [TSC](#technical-steering-committee-tsc) members and Core members.

Author: ```suggestion
```
